### PR TITLE
{lang}[intel/2015b] Python 3.5.0 + several pkgs on top of it (WIP)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (7 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/92c2fb2ad3c012734698 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/21622ad0a19d51c6fb44 for a full test report.

File: easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015b-Python-3.5.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+python = 'Python'
+pythonversion = '3.5.0'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('freetype', '2.5.5'),
+    ('libpng', '1.6.16'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015b-Python-3.5.0.eb]: update to the latest and greatest? (for both)

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015b-Python-3.5.0.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015b-Python-3.5.0.eb]: same matplotlib version as existing ones, I won't bump these versions until there's a newer version of matplotlib

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**